### PR TITLE
HC-937_Junior accounts

### PR DESCRIPTION
**Pull request by Renat0Ribeir0**
**Changes proposed in this pull request**
- This article explains the differences between Junior and Adult accounts for users between 14 and 16 years old and describes the upgrading process from a Junior to an Adult account.

### CODE REVIEW

File: content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+<h2 id="Adult-accounts"></h2>
+
+## Adult accounts for users between 14 and 16 years old
+
+As for the Junior Accounts, users between 14 and 16 years old will be asked to provide the email of the person exercising parental authority. Arduino will process the [personal data](https://www.arduino.cc/en/privacy-policy/children-privacy-policy) of the latter.
+
+Through the Adult Account for users between 14 and 16 years old, the user has access to all the services available on the Arduino website, except for those services/features expressly reserved for the person exercising parental authority. Only the person exercising parental authority may choose which services would be activated and which services the user between 14 and 16 years old can access.
+
+## Useful information
```
Comment (seaxwi) [inline content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md:42]:
```suggestion
## Further reading
```

File: content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+title: "About Arduino accounts for minors"
+---
+
+Arduino offers two types of accounts for minors: <a href="#Junior-accounts">Junior accounts</a> and <a href="#Adult-accounts">Adult accounts for users between 14 and 16 years old</a>. They differ in the features that users can access and the way that Arduino handles [personal data](https://www.arduino.cc/en/privacy-policy/children-privacy-policy). Both account types require the minor's account to be linked to a person exercising parental authority. Learn more about Arduino accounts for minors below.
```
Comment (seaxwi) [inline content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md:5]:
```suggestion
Arduino offers two types of accounts for minors: [Junior accounts](#Junior-accounts) and [Adult accounts for users between 14 and 16 years old](#Adult-accounts). They differ in the features that users can access and the way that Arduino handles [personal data](https://www.arduino.cc/en/privacy-policy/children-privacy-policy). Both account types require the minor's account to be linked to a person exercising parental authority. Learn more about Arduino accounts for minors below.
```
Markdown syntax works with anchor links.

File: content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+### Upgrading to an Adult account
+
+After turning 14, users of Junior accounts will be prompt to upgrade to an <a href="#Adult-accounts">Adult account for users between 14 and 16 years old</a>.
+
+![Upgrading to regular account pop-up](img/Upgrade_to_regular_account.png)
+
+This process will move all their files, projects & data to the upgraded account, but the account will still require a person exercising parental authority.
+
+<h2 id="Adult-accounts"></h2>
```
Comment (seaxwi) [inline content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md:34]:
```suggestion
<a id="Adult-accounts"></a>
```

File: content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+---
+title: "About Arduino accounts for minors"
+---
+
+Arduino offers two types of accounts for minors: <a href="#Junior-accounts">Junior accounts</a> and <a href="#Adult-accounts">Adult accounts for users between 14 and 16 years old</a>. They differ in the features that users can access and the way that Arduino handles [personal data](https://www.arduino.cc/en/privacy-policy/children-privacy-policy). Both account types require the minor's account to be linked to a person exercising parental authority. Learn more about Arduino accounts for minors below.
+
+<h2 id="Junior-accounts"></h2>
```
Comment (seaxwi) [inline content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md:7]:
```suggestion
<a id="Junior-accounts"></a>
```
I'm not sure if either is considered good practise but this is what I've been using to add "anchor targets".

File: content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md (hunk 2 above)
Comment (seaxwi) [reply content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md:5]: * Markdown syntax works with anchor links.
* For me the first sentance was hard to follow, I would suggest adding some structure:

```suggestion
Arduino offers two types of accounts for minors:

* [Junior accounts](#Junior-accounts). For users below 14 years old.
* [Adult accounts with restrictions](#Adult-accounts). For users between 14 and 16 years old.

They differ in the features that users can access and the way that Arduino handles [personal data](https://www.arduino.cc/en/privacy-policy/children-privacy-policy). Both account types require the minor's account to be linked to a person exercising parental authority. Learn more about Arduino accounts for minors below.
```

File: content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+* Access Educational products and services intended for middle school students (view the full list on [Arduino Education](https://www.arduino.cc/education)).
+* Download and use Arduino IDE.
+* Download and use the Arduino Science Journal App. Use the Science Journal Apps only for creating experiments saved locally in the App. Sharing functionality with Google Drive is not made available to Junior Accounts.
+* Submit Assignment from the Science Journal App to Google Classroom, subject to an existing plan held by your teacher and approval of the latter.
+
+### Upgrading to an Adult account
+
+After turning 14, users of Junior accounts will be prompt to upgrade to an <a href="#Adult-accounts">Adult account for users between 14 and 16 years old</a>.
+
+![Upgrading to regular account pop-up](img/Upgrade_to_regular_account.png)
```
Comment (seaxwi) [inline content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md:30]: The image is very large, I would scale it down to 600 px width.

File: content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+This process will move all their files, projects & data to the upgraded account, but the account will still require a person exercising parental authority.
```
Comment (seaxwi) [inline content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md:32]:
```suggestion
This will transfer all their files, projects and data to the upgraded account, but the account will still require a person exercising parental authority.
```

File: content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+<h2 id="Adult-accounts"></h2>
+
+## Adult accounts for users between 14 and 16 years old
+
+As for the Junior Accounts, users between 14 and 16 years old will be asked to provide the email of the person exercising parental authority. Arduino will process the [personal data](https://www.arduino.cc/en/privacy-policy/children-privacy-policy) of the latter.
+
+Through the Adult Account for users between 14 and 16 years old, the user has access to all the services available on the Arduino website, except for those services/features expressly reserved for the person exercising parental authority. Only the person exercising parental authority may choose which services would be activated and which services the user between 14 and 16 years old can access.
```
Comment (seaxwi) [inline content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md:40]:
```suggestion
With an Adult Account for users between 14 and 16 years old, the user has access to all the services available on the Arduino website, except for those services and features expressly reserved for the person exercising parental authority. Only the person exercising parental authority may choose which services would be activated and which services the user between 14 and 16 years old can access.
```

**Review by seaxwi — CHANGES_REQUESTED** [review]

File: content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
+---
+title: "About Arduino accounts for minors"
+---
+
+Arduino offers two types of accounts for minors:
+
+* [Junior accounts](#Junior-accounts). For users below 14 years old.
+* [Adult accounts with restrictions](#Adult-accounts). For users between 14 and 16 years old.
```
Comment (seaxwi) [inline content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md:8]: Perhaps we should call them "regular" accounts instead of "adult".

This is how they're referred to in the upgrade prompt, and it makes sense since 14 year olds are not adults.

image

**Review by seaxwi — CHANGES_REQUESTED** [review]

File: content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md (hunk 8 above)
Comment (Renat0Ribeir0) [reply content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md:8]: Yes, I agree, it is less confusing that way since the image calls it a "regular account". My only concern is that in Arduino's privacy policy, these accounts are referred to as "Adult account (for users between 14 and 16 years old" and this can create confusion.

Comment (Renat0Ribeir0) [reply content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md]: Updated [b55cbb9](https://github.com/arduino/help-center-content/pull/62/commits/b55cbb91f85b8f5c489bf267b539df5efaf7527a)

Comment (seaxwi) [reply content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md]: @Renat0Ribeir0 Ah, then I see where you were coming from. But yeah on the whole I think it's a change for the better.

**Review by seaxwi — APPROVED** [review]
